holmes/plugins/toolsets/kafka.py

```diff
@@ -352,17 +382,62 @@ def _invoke(self, params: Dict) -> StructuredToolResult:
             )
 
     def get_parameterized_one_liner(self, params: Dict) -> str:
-        return f"Described topic: {params['topic_name']} in cluster {params['kafka_cluster_name']}"
+        return f"Described topic: {params['topic_name']} in cluster \"{params.get('kafka_cluster_name')}\""
 
 
 def group_has_topic(
-    consumer_group_description: ConsumerGroupDescription, topic_name: str
+    client: AdminClient,
+    consumer_group_description: ConsumerGroupDescription,
+    topic_name: str,
+    bootstrap_servers: str,
 ):
+    # Check active member assignments
     for member in consumer_group_description.members:
-        if len(member.assignment.topic_partitions) > 0:
-            member_topic_name = member.assignment.topic_partitions[0].topic
-            if topic_name == member_topic_name:
+        for topic_partition in member.assignment.topic_partitions:
+            if topic_partition.topic == topic_name:
+                return True
+
+    # Check committed offsets for the topic (handles inactive/empty consumer groups)
+    try:
+        # Try using the Consumer class to check committed offsets for the specific group
+
+        # Create a consumer with the same group.id as the one we're checking
+        # This allows us to check its committed offsets
+        consumer_config = {
+            "bootstrap.servers": bootstrap_servers,
+            "group.id": consumer_group_description.group_id,
+            "auto.offset.reset": "earliest",
+            "enable.auto.commit": False,  # Don't auto-commit to avoid side effects
+        }
+        consumer = Consumer(consumer_config)
+
+        # Get topic metadata to know which partitions exist
+        topic_metadata = client.list_topics(topic_name, timeout=10)
```

Comment:
I think we can call list_topics and create the topic_partitions only once and get it as a parameter to group_has_topic, saving api calls to client.list_topics.